python/tvm/driver/tvmc/autotuner.py

```diff
@@ -210,6 +210,13 @@ def add_tune_parser(subparsers):
     #     can be improved in future to add integration with a modelzoo
     #     or URL, for example.
     parser.add_argument("FILE", help="path to the input model file")
+    parser.add_argument(
+        "--shapes",
```

Comment:
Question: is the terminology `--shapes` specific enough for people to always infer they are "input shapes"? I'm asking because in the PR we previously had in discussion, we called it `--input-shapes`, and just want to hear what others think.

cc @comaniac @ekalda

Reply:
input-shapes is better.